bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/HHAClaimGroup.java

```diff
@@ -1,1173 +1,1176 @@
-package gov.hhs.cms.bluebutton.datapipeline.rif.model;

```

Comment:
Looks like our line-ending fix has us in for some pain until we go through and fix every file that was committed with `CRLF` line endings. (Git is now converting those on commit to Unix line endings.)

Can you please file a bug for me in Sprint 11 to go through and do all of these at once? Diffs will be useless on many files for a while unless/until we do.